api/HTMLTrackElement.json

```diff
@@ -74,24 +91,74 @@
             "ie": {
               "version_added": false
             },
-            "opera": {
-              "version_added": "15"
-            },
-            "opera_android": {
-              "version_added": "14"
-            },
-            "safari": {
-              "version_added": "7"
-            },
-            "safari_ios": {
-              "version_added": "7"
-            },
-            "samsunginternet_android": {
-              "version_added": "1.5"
-            },
-            "webview_android": {
-              "version_added": "≤37"
-            }
+            "opera": [
+              {
+                "version_added": "19"
+              },
+              {
+                "version_added": "15",
+                "partial_implementation": true,
+                "notes": "The <code>oncuechange</code> event handler property is not supported."
+              },
+              {
+                "version_added": "12",
```

Comment:
Yep, confirmed with http://software.hixie.ch/utilities/js/live-dom-viewer/?saved=10032 for Opera 12.16.